modules/core/src/main/scala/qq/droste/data/Mu.scala

```diff
@@ -13,7 +13,11 @@ import cats.syntax.functor._
   * In Haskell this can more aptly be expressed as:
   * `data Mu f = Mu (forall x . (f x -> x) -> x)`
   */
-sealed abstract class Mu[F[_]] extends (Algebra[F, ?] ~> Id) with Serializable
+sealed abstract class Mu[F[_]] extends Serializable {
+  def apply[A](fold: Algebra[F, A]): A
+
+  def toFunctionK: Algebra[F, ?] ~> Id = λ[Algebra[F, ?] ~> Id](Mu.this.apply(_))
```

Comment:
There is probably a better name for this. `algebraFunctionK`? Everything that I could think of sounded gross to me.

Reply:
To FunctionK seems fine to me.